test/integration/smartCache.js

```diff
@@ -0,0 +1,28 @@
+const app = require('../../server');
+const constants = require('../../app/lib/constants');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+describe('smartCache', () => {
```

Comment:
I know the name of the module is `smartCache` but if that changes this also needs to change (along with this file name). In order to keep the tests valid through a renaming of the module if this was `describe('Cache-Control headers', ... )` it wouldn't need to change.